### PR TITLE
Added vm sku policy definition

### PR DESCRIPTION
**Pull request by QBY-MarkusMaring**
# Description

This PR will define a policy to deny certain VM SKUs. This policy is not assigned by default.

# Change overview (tick true):

- [ ] This introduces backward incompatible changes
- [x] This adds a new backward compatible Feature
- [ ] This fixes a Bug

# Version information: 

- Previous Version: `2.3.0`

- Next Version based on [Semantic Versioning](https://semver.org/#summary) (see above): `3.0.0`

# How Has This Been Tested?

- [x] Terraform apply of CAF in test enironment

# Checklist:

- [x] I have run tests and documented them above
- [x] I have performed a self-review of my own code
- [ ] I have updated the documentation
- [x] I have updated the CHANGELOG

### CODE REVIEW

**Review by QBY-ThimoLimpert — APPROVED** [review]
Nice Work!

File: policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
+    "name": "QBY-Deny-VM-SKUs",
```
Comment (QBY-ThimoLimpert) [inline policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json:2]:
```suggestion
    "name": "QBY-Allowed-VM-SKUs",

```

File: policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{
+    "name": "QBY-Deny-VM-SKUs",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "properties": {
+        "displayName": "Deny VM SKUs",
```
Comment (QBY-ThimoLimpert) [inline policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json:5]:
```suggestion
        "displayName": "Allowed VM/VMSS SKUs",

```

**Review by QBY-ThimoLimpert — CHANGES_REQUESTED** [review]
This Policy shoudn't be a deny Policy, but a allow policy.

**Review by QBY-ChristianHartmann — APPROVED** [review]
great work

File: policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            "listOfAllowedSKUs": {
+                "type": "Array",
+                "metadata": {
+                    "description": "The list of size SKUs that can be used for virtual machines. Items may contain wildcards.",
+                    "displayName": "Allowed Size SKUs"
+                }
+            },
+            "effect": {
+                "type": "String",
+                "defaultValue": "DeployIfNotExists",
```
Comment (QBY-ThimoLimpert) [inline policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json:25]:
```suggestion
                "defaultValue": "Deny",

```

File: policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                "allowedValues": [
+                    "AuditIfNotExist",
+                    "DeployIfNotExists",
```
Comment (QBY-ThimoLimpert) [inline policy_definitions/vmskus/policy_definition_qby_deny_vm_skus.json:28]:
```suggestion
                    "Audit",
                    "Deny",

```

**Review by QBY-ThimoLimpert — APPROVED** [review]

**Pull request merged**